snapshots/hmd/2024-11-19/hfd.py

```diff
@@ -0,0 +1,25 @@
+"""To be able to download the dataset, you need to be registered at HFD."""
```

Comment:
You can maybe add more specific instructions, such as the link and the file to download